client/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "vite build -m production",
     "prebuild": "node ./scripts/build-regions.js",
     "postinstall": "pnpm prebuild",
-    "test:run": "size-limit",
+    "test:run": "node --test test/*.test.js && size-limit",
```

Comment:
We are using new way to run all these tests in parallel:

https://github.com/browserslist/browsersl.ist/blob/main/package.json#L14-L18

Let’s replace `&&` to two commands with `test:` prefix (and replace `test` script here to [regexp](https://github.com/hplush/slowreader/blob/main/web/package.json#L12)).